beetsplug/fetchart.py

```diff
@@ -204,6 +236,8 @@ def resize(self, plugin):
                 max_filesize=plugin.max_filesize,
             )
         elif self.check == self.CANDIDATE_DOWNSIZE:
+            assert self.size is not None
```

Comment:
But this is about fetchart only computing the size if it really needs to (because doing so means calling a subprocess, which requires the appropriate dependency installed and is potentially slow). So, the typing part of this is also up to fetchart. However, this is gone now due a refactoring.